spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-functional-binder-sample/sink/src/main/resources/application.properties

```diff
@@ -0,0 +1,10 @@
+spring.cloud.stream.bindings.convertUserMessageToString-in-0.destination=[topic]
+
+spring.cloud.function.definition=convertUserMessageToString
+
+# If group is specified, the Pub/Sub subscription name will be [PUBSUB_TOPIC_NAME].[PUBSUB_GROUP_NAME]
+spring.cloud.stream.bindings.input.group=[subscription]
```

Comment:
Should this be `spring.cloud.stream.bindings.convertUserMessageToString-in-0.group`? not 'input'

Reply:
Same reason. We don't get to use shortcut input/output names because of that zombie scheduler bean hanging around the context, making it look like two eligible functions.

Reply:
Not following. What I am saying is that the binding name is `convertUserMessageToString-in-0` not 'input'. In other words the above property has no effect in the current config 
no matter wha the value is

Reply:
Oh, that's right; it's the wrong binding name, so anonymous group will be used. I'll fix it.
I thought you were asking why a simple `spring.cloud.stream.bindings.input.destination` could not be used instead of `spring.cloud.stream.bindings.convertUserMessageToString-in-0.destination`.

Reply:
Also, if you do want to use more descriptive names you can use `spring.cloud.stream.function.bindings.convertUserMessageToString-in-0=input`. With that you just mapped generated name to `input` so from that point you can refer to binding name as `input`.